src/Klarna/Rest/Resource.php

```diff
@@ -236,7 +247,12 @@ protected function delete($url, array $data = null)
      */
     protected function patch($url, array $data)
     {
-        return $this->request('PATCH', $url, ['Content-Type' => 'application/json'], json_encode($data));
+        return $this->request(
+            'PATCH',
```

Comment:
Use constant from Method Class

Reply:
Replaced all the text methods in all test files.